packages/stepper/src/EllipsesStep/EllipsesStep.styles.ts

```diff
@@ -0,0 +1,35 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { spacing } from '@leafygreen-ui/tokens';
+
+import { stepIconClassName } from '../constants';
+
+import { EllipsesStepState, EllipsesStepStates } from './EllipsesStep.types';
+
+// TODO: would be good to put these styles inside a baseStyles, but it is currently impossible because the <Tooltip> content is an iframe.
```

Comment:
iframe?

Reply:
Hahahah this is Sean on his first project being confused. Someone probably explained the Portal in Tooltip as "like an iframe" or something and I just soaked it up